### PR TITLE
v3.0, dev: add tidb-in-kubernetes docs

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

add tidb-in-kubernetes docs

### Which version does your change affect? 

v3.0, dev

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by yikeke** [comment]
PTAL @lilin90

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# Deploy TiDB Operator and TiDB Cluster on Alibaba Cloud Kubernetes
+---
+title: Deploy TiDB on Alibaba Cloud Kubernetes
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
title: Deploy TiDB on Alibaba Cloud
```

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+summary: Learn how to deploy a TiDB cluster on Alibaba Cloud Kubernetes.
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
summary: Learn how to deploy a TiDB cluster on Alibaba Cloud.
```

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+category: how-to
+---
 
-This document introduces how to use your laptop (Linux or macOS) to deploy TiDB Operator and TiDB Cluster on Alibaba Cloud Kubernetes.
+# Deploy TiDB on Alibaba Cloud Kubernetes
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
# Deploy TiDB on Alibaba Cloud
```

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+
+This document describes how to deploy a TiDB cluster on Alibaba Cloud Kubernetes with your laptop (Linux or macOS) for development or testing.
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
This document describes how to deploy a TiDB cluster on Alibaba Cloud with your laptop (Linux or macOS) for development or testing.
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+    > **Note:**
+    > 
+    > The access key used must be granted permissions to control resources.
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.12
+- [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.1 and <= 2.11.0
+- [jq](https://stedolan.github.io/jq/download/) >= 1.6
+- [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) 0.11.*
+
+### Permissions
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
### Privileges
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+The following permissions are required to deploy a TiDB cluster:
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
To deploy a TiDB cluster, make sure you have the following privileges: 
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+You can then interact with the ACK cluster using `kubectl` and `helm` (`cluster_name` is `tidb-cluster` by default):
+
+```shell
+$ export KUBECONFIG=$PWD/credentials/kubeconfig_<cluster_name>
+$ kubectl version
+$ helm ls
+```
+
+## Access the DB
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
## Access the cluster
```

**Pull request merged**